css/project-page.styl

```diff
@@ -1,3 +1,22 @@
+.project-about-page
+  background: rgba(255,255,255,0.95)
+  border-radius: 3px
+  color: #000000
+  margin: 1.5em 12vw
+  padding: 1.5em 6vw
+
+  .markdown
+    img
+      max-width: 100%
+
+    h1:first-child, h2:first-child,
```

Comment:
That's a bit tricky. If using `:first-of-type`, multiple headers might be colored orange. For example, if the markdown has an `h1` and `h6` in the same section. To my understanding, `.first-child` only targets the first item in that class, which might be more fitting for the template.